aggregator/entry.go

```diff
@@ -140,7 +140,7 @@ func (e *Entry) AddMetricWithPoliciesList(
 	switch mu.Type {
 	case unaggregated.BatchTimerType:
 		var err error
-		if err = e.applyRateLimit(int64(len(mu.BatchTimerVal))); err == nil {
+		if err = e.applyValueRateLimit(int64(len(mu.BatchTimerVal))); err == nil {
```

Comment:
It identifies the thing that we are applying rate limiting to. Feels like `frequency` is already captured by the "rate" in "rate limit", no?

Reply:
value feels like the value of the metric rather than the length of the timer/metric batch though, can it just be errWriteRateLimitExceeded and the new metrics one be errNewMetricRateLimit...?